apps/guardian-ui/src/components/Wrapper.tsx

```diff
@@ -1,5 +1,7 @@
 import React, { memo } from 'react';
-import { Box, Center } from '@chakra-ui/react';
+import { Box, Flex } from '@chakra-ui/react';
+import { Header } from './Header';
+import Footer from './Footer';
```

Comment:
nit: we should probably standardize our component exports. Non -

Reply:
Alright, will make necessary corrections on the next issue